scripts/gradle/projectDependencyGraph.gradle

```diff
@@ -0,0 +1,116 @@
+task projectDependencyGraph {
```

Comment:
All the `gradle` files are migrated to `kotlin dsl` in order to keep project consistency maybe we should do the same with this one. I found an already [one](https://github.com/vanniktech/gradle-dependency-graph-generator-plugin/blob/1c065cb50041b911effb6b2c61406242a33ac8d4/src/main/kotlin/com/vanniktech/dependency/graph/generator/ProjectDependencyGraphGenerator.kt),  as a good start point.

The path should be `buildSrc/.../plugins` or `buildSrc/.../utils` if you consider to migrate to kotlin. The `scripts` directory was created for bash scripts not for gradle tasks.

Reply:
OK, Good point, I was already looking for one, I didn't find that. I will check it